src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/Strategy.kt

```diff
@@ -36,4 +36,5 @@ abstract class Strategy protected constructor(
      * Is invoked before each actor execution.
      */
     open fun onActorStart(iThread: Int) {}
+    open fun beforeValidationFunctionCall() {}
```

Comment:
Please add documentation, indicating when this function should be called (and why)